apps/dashboard/src/components/subscribers/preferences/subscriber-preferences-toggles.tsx

```diff
@@ -0,0 +1,45 @@
+import { STEP_TYPE_TO_ICON } from '@/components/icons/utils';
```

Comment:
Nit-picky comment: The file naming is a bit verbose. I suggest renaming this to `apps/dashboard/src/components/subscribers/preference-list-item.tsx`. 

Also currently we render it with a toggle. If we decide to use a checkbox, we shouldn't need to rename the file to something-checkbox ;).